### ISSUE TITLE
Create mental model analogy/ illo for getting started section

### ISSUE THREAD

**Issue by dcossyleon** [dcossyleon/firstclasspackage#4]

**Comment by apreshill** [comment]
I have a possible analogy for the data part. I love this image btw:

![](https://rstudio4edu.github.io/firstclasspackage/images/datapackagemap.jpg)

Bear with me here as it may be kind of grizzly :sushi:

- :fishing_pole_and_fish: `data_raw/`: this is your fish

- :sushi: `data/`: this is your sushi roll (re: some additional data prep here like wrangling, and final object gets "rolled up" into a nice clean `.rda` file)

- :memo: `R/`: write up your menu- what is inside of your roll?

- :bento: `man/`: this is you plating/serving the roll 

Does this analogy work? Might it be helpful?

**Comment by dcossyleon** [comment]
haha, yes! I like it a lot. I think the analogies for `data_raw/`, `data/`, and `R/` are on point. I think of `man/` as not really containing the data itself, but just providing the nicer documentation and context. So maybe a better analogy for this is the table-side explanation by the chef or server of what the roll is and what it pairs well with, etc. etc. This analogy could lend itself well to some great illos!

**Comment by apreshill** [comment]
You are right- maybe `man/` is printing out the menu? Since it comes directly from `.R` (i.e., the one-to-one correspondence). Then the bento box at the end is your package- the result of devtools document/check/install--> WHALA

**Comment by apreshill** [comment]
I like the chef take too- because it is a bit higher level- I think of this Portlandia episode haha:

https://www.youtube.com/watch?v=G__PVLB8Nm4

**Comment by apreshill** [comment]
To continue on the theme, here are some 🐟 datasets we could use to build the data package:

+ https://www.pnas.org/content/114/7/1512/tab-figures-data ([download](https://www.pnas.org/highwire/filestream/29164/field_highwire_adjunct_files/1/pnas.1617948114.sd01.xlsx))
+ http://users.stat.ufl.edu/~winner/data/fishermen_mercury.txt ([download](http://users.stat.ufl.edu/~winner/data/fishermen_mercury.csv))
+ https://fishsciences.github.io/post/visualizing-fish-encounter-histories/ ([built into tidyr package](https://github.com/tidyverse/tidyr/blob/master/data-raw/fish_encounters.R))
+ https://en.wikipedia.org/wiki/Fishing_industry_by_country ([built into Mine's dsbox package](https://github.com/rstudio-education/dsbox/blob/master/data-raw/fisheries.R))
